### ISSUE TITLE
welcome!

### ISSUE THREAD

**Issue by fgregg** [fgregg/chicago-historical-addresses#4]
glad you are here! please respond to this issue with a little introduction and how you hope to contribute to te project

**Comment by fgregg** [comment]
i’m forest gregg. i’m a chicago data person. i’m hoping to do data entry and help with coordination

**Comment by tewhalen** [comment]
hi all, i'm ted whalen, dabbler in both tech and local history. hoping to help with automated ocr and thinking through how the data will eventually be used.

thanks for setting this up, @fgregg!

**Comment by leylabauer** [comment]
Hi! I'm Leyla, I set up the [http://www.chicagoaddressconversionguide.com/](url) project out of a bootcamp back in 2019 but did not complete it due to roadblocks on 'how is the best/fastest way to get this data organized'. Here to help with others and gain more insight on the best way to answer the question above and help create an easily-accessible database of address that's open to whoever needs to use it. Otherwise, local Chicago twitter person, Chicago history tour guide, and data ETL person.

**Comment by gneidhardt** [comment]
Hi, I'm Gretchen Neidhardt, and I'm a the cataloging and metadata librarian at the Chicago History Museum. I'm a frequent user of address/street name datasets, and I'd love to see a more searchable (and possibly integrated) version of them. I have a lot of experience in coordinating manual transcription for searchable datasets (mostly data that then gets transformed into library catalog records, or metadata assigned to objects like images). I'm happy to contribute thoughts regarding public usability and hosting, and help volunteer and coordinate data review. I am not as well versed yet in database administration, coding, or the automation that goes into building this set, but I'd like to be. 

I'm also wondering if there might be potential here for a Wikidata project, at least for tracking the alternative street names, since many current streets already exist as entities, and an alternative name statement with time properties could be added? This might be outside the scope of this project, but I wasn't sure if perhaps building out the street name change data there could be drawn into the address change database. Apologies if this is taking the conversation in too much of a different direction, but while I know the library world uses Wikidata quite a bit, I'm not sure if it's used as much elsewhere.

EDIT: Looks like Rogers Park/West Ridge Historical Society might have already started some of this info on street name changes and dates: https://rpwrhs.org/w/index.php?title=Howard_Avenue

**Comment by tewhalen** [comment]
@gneidhardt i would very much like to see this effort be able to get connected to street renamings, and I hope that however we end up transcribing this data, we capture everything that could be valuable. Maybe it would be useful to start a thread talking about potential use cases?

**Comment by gneidhardt** [comment]
@tewhalen yes! I added something like that to the readme section, but happy to start another thread. quoting my other response below:

"Chicago History users would definitely like to have a searchable database, where they enter a current or old address and the reverse comes up. I think ideally they'd also like to know if that street has a different name (and when that changed), possibly where the name came from, and links to PIN, building permit, etc. UIC is currently undergoing a project to transcribe building permits and I can see if someone there wants to be involved here?

As Robert mentioned via Twitter, I think the researcher ideal would be an interactive map with all that information."

**Comment by aoifefahey** [comment]
hi, I'm aoife. i've spent most of my career as a data person for nonprofits, specializing in fundraising & marketing ( and i'm currently looking for a role if anyone has any leads). i have a fairly substantial amount of experience designing & implementing data entry systems as a result, though that experience doesn't necessarily translate well unless we choose to enter most of the the data manually.

my most notable contributions to chicago twitter are asking the question that kicked this project off, and creating an L map of chicago where every station is named western

**Comment by gneidhardt** [comment]
Hey all - I don't think there's been too much action on this recently, but I wanted to let folks know that as of Jan 10, I'll be in a new position at Northwestern's medical school library. I'm still really interested in this project and more than happy to advise and/or volunteer, but I can also bring in someone more officially affiliated with Chicago History Museum, if you'd like to keep that connection going. I certainly think no matter what they'd be a great place to garner volunteers and publicity (when needed). I am not sure that they'd be the best for hosting something digital, but I know they'd certainly link to it (and maybe I'm wrong in that hosting assessment). I know all my colleagues in the Research Center there were just thrilled that this was getting discussed since unfortunately we do not have anyone on staff with the skills to facilitate this type of project on the back end. Anyway, I'll still lurk if it's cool, but def feel free to reach out if you'd like me to make any intros!
